site/source/docs/building_from_source/verify_emscripten_environment.rst

```diff
@@ -17,7 +17,7 @@ The first step in verifying the environment is to run Emscripten with the versio
 
 Open a terminal in the directory in which you installed Emscripten (on Windows open the :ref:`Emscripten Command Prompt <emcmdprompt>`). Then call the :ref:`Emscripten Compiler Frontend (emcc) <emccdoc>` as shown::
 
-  ./emcc -v
+  emcc -v
```

Comment:
These ones should have the `./` in them as hey explicitly talk about opening a terminal in the directory where emscripten is installed..  without the `./` the emc cin the current working directory might not be the one being run.

There are quite a few cases like this.   I think its best to revert this change and try to re-land a narrower version.

Reply:
> There are quite a few cases like this. I think its best to revert this change and try to re-land a narrower version.

Mh =( Unfortunately I have no more time and motivation currently so I think I'll accept this and leave it as is - thanks for the support so far.